src/main/scala/com/github/swagger/akka/SwaggerHttpService.scala

```diff
@@ -46,15 +46,15 @@ trait SwaggerGenerator {
   def basePath: String = "/"
   def apiDocsPath: String = "api-docs"
   def info: Info = Info()
-  def scheme: Scheme = Scheme.HTTP
+  def schemes: List[Scheme] = List(Scheme.HTTP)
```

Comment:
This change breaks compatibility for existing users.
We can either support scheme and schemes (ignore the scheme if the schemes is empty) or we can make the breaking change and release it as a new 0.11 version.